.travis.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - pip install -e ".[test]"
     - pip check
     script:
-    - python3 -m flake8
+    - python3 -m flake8 --max-complexity 10
```

Comment:
we can increase amount if necessary ) just for notice
```This practice was adopted by the NIST Structured Testing methodology, with an observation that since McCabe's original publication, the figure of 10 had received substantial corroborating evidence, but that in some circumstances it may be appropriate to relax the restriction and permit modules with a complexity as high as 15.```

Reply:
Most of the code can by easily refactored ... except `Cluster.topology_data` which is a nightmare.